Assets/MXR.SDK/Runtime/Android/Utils/SafeJNI.cs

```diff
@@ -0,0 +1,186 @@
+﻿using System;
+
+using UnityEngine;
+
+namespace MXR.SDK {
+    /// <summary>
+    /// This utility class provides extension methods for invoking JNI methods while logging any errors that occur.
+    /// The occurring exception is consumed and not propagated up, this is intentional as our general usage of JNI 
+    /// in Unity C# is to only check if a call succeeded or not, and not really handle exceptions.
+    /// </summary>
+    public static class SafeJNI {
+        const string TAG = "SafeJNI";
+
+        /// <summary>
+        /// Calls a void-returning static method on a native object using method name and optional arguments. 
```

Comment:
Is this comment out dated ? `void-returning`

Reply:
Improved the language: https://github.com/ManageXR/mxr-unity-sdk/pull/61/commits/dfd15b2dc265af9d01d1d466e5fe4a313e7230e7